Formula/schemathesis.rb

```diff
@@ -0,0 +1,198 @@
+  end
+  resource "charset_normalizer" do
+    url "https://files.pythonhosted.org/packages/93/1d/d9392056df6670ae2a29fcb04cfa5cee9f6fbde7311a1bb511d4115e9b7a/charset-normalizer-2.1.0.tar.gz"
+    sha256 "575e708016ff3a5e3681541cb9d79312c416835686d054a23accb873b254f413"
+  end
+  resource "urllib3" do
+    url "https://files.pythonhosted.org/packages/6d/d5/e8258b334c9eb8eb78e31be92ea0d5da83ddd9385dc967dd92737604d239/urllib3-1.26.11.tar.gz"
+    sha256 "ea6e8fb210b19d950fab93b60c9009226c63a28808bc8386e05301e25883ac0a"
+  end
+  resource "certifi" do
+    url "https://files.pythonhosted.org/packages/cc/85/319a8a684e8ac6d87a1193090e06b6bbb302717496380e225ee10487c888/certifi-2022.6.15.tar.gz"
+    sha256 "84c85a9078b11105f04f3036a9482ae10e4621616db313fe045dd24743a0820d"
+  end
+  resource "iniconfig" do
+    url "https://files.pythonhosted.org/packages/23/a2/97899f6bd0e873fed3a7e67ae8d3a08b21799430fb4da15cfedf10d6e2c2/iniconfig-1.1.1.tar.gz"
+    sha256 "bc3af051d7d14b2ee5ef9969666def0cd1a000e121eaea580d4a313df4b37f32"
+  end
+  resource "pluggy" do
+    url "https://files.pythonhosted.org/packages/a1/16/db2d7de3474b6e37cbb9c008965ee63835bba517e22cdb8c35b5116b5ce1/pluggy-1.0.0.tar.gz"
+    sha256 "4224373bacce55f955a878bf9cfa763c1e360858e330072059e10bad68531159"
+  end
+  resource "py" do
+    url "https://files.pythonhosted.org/packages/98/ff/fec109ceb715d2a6b4c4a85a61af3b40c723a961e8828319fbcb15b868dc/py-1.11.0.tar.gz"
+    sha256 "51c75c4126074b472f746a24399ad32f6053d1b34b68d2fa41e558e6f4a98719"
+  end
+  resource "sniffio" do
+    url "https://files.pythonhosted.org/packages/a6/ae/44ed7978bcb1f6337a3e2bef19c941de750d73243fc9389140d62853b686/sniffio-1.2.0.tar.gz"
+    sha256 "c4666eecec1d3f50960c6bdf61ab7bc350648da6c126e3cf6898d8cd4ddcd3de"
+  end
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+
+    # Install hypothesis_graphql and graphql-core using brewed poetry to avoid build dependency on Rust.
+    resource("hypothesis_graphql").stage do
+      system Formula["poetry"].opt_bin/"poetry", "build", "--format", "wheel", "--verbose", "--no-interaction"
+      venv.pip_install_and_link Dir["dist/hypothesis_graphql-*.whl"].first
+    end
+    resource("graphql-core").stage do
+      system Formula["poetry"].opt_bin/"poetry", "build", "--format", "wheel", "--verbose", "--no-interaction"
+      venv.pip_install_and_link Dir["dist/graphql_core-*.whl"].first
+    end
+    # Install jsonschema using hatch, to avoid re-installing it
+    resource("jsonschema").stage do
+      system Formula["hatch"].opt_bin/"hatch", "build", "--target", "wheel"
+      venv.pip_install_and_link Dir["dist/jsonschema-*.whl"].first
+    end
+    # Install exceptiongroup using flit, to avoid re-installing it
+    resource("exceptiongroup").stage do
+      system Formula["flit"].opt_bin/"flit", "build", "--format", "wheel"
+      venv.pip_install_and_link Dir["dist/exceptiongroup-*.whl"].first
```

Comment:
```suggestion
      venv.pip_install_and_link Pathname.glob("dist/hypothesis_graphql-*.whl").first
    end
    resource("graphql-core").stage do
      system Formula["poetry"].opt_bin/"poetry", "build", "--format", "wheel", "--verbose", "--no-interaction"
      venv.pip_install_and_link Pathname.glob("dist/graphql_core-*.whl").first
    end
    # Install jsonschema using hatch, to avoid re-installing it
    resource("jsonschema").stage do
      system Formula["hatch"].opt_bin/"hatch", "build", "--target", "wheel"
      venv.pip_install_and_link Pathname.glob("dist/jsonschema-*.whl").first
    end
    # Install exceptiongroup using flit, to avoid re-installing it
    resource("exceptiongroup").stage do
      system Formula["flit"].opt_bin/"flit", "build", "--format", "wheel"
      venv.pip_install_and_link Pathname.glob("dist/exceptiongroup-*.whl").first
```